en/deploy-tidb-from-kubernetes-gke.md

```diff
@@ -257,64 +191,53 @@ Congratulations, you are now up and running with a distributed TiDB database com
 
-{{< copyable "shell-regular" >}}
-
 ``` shell
 kubectl -n demo edit tc basic
 ```
 
 ## Access the Grafana dashboard
 
-To access the Grafana dashboards, you can create a tunnel between the Grafana service and your shell.
-To do so, use the following command:
+To access the Grafana dashboards, you can forward a port from the Cloud Shell to the Grafana service in Kubernetes. (Cloud Shell already uses port 3000 so we use port 3003 in this example instead.)
 
-{{< copyable "shell-regular" >}}
+To do so, use the following command:
 
 ```shell
-kubectl -n demo port-forward svc/basic-grafana 3000:3000 &>/dev/null &
+kubectl -n demo port-forward svc/basic-grafana 3003:3000 &>/tmp/pf3003.log &
 ```
 
-In Cloud Shell, click on the Web Preview button and enter 3000 for the port. This opens a new browser tab pointing to the Grafana dashboards. Alternatively, use the following URL <https://ssh.cloud.google.com/devshell/proxy?port=3000> in a new tab or window.
+Open this URL to view the Grafana dashboard: <https://ssh.cloud.google.com/devshell/proxy?port=3003> . (Alternatively, in Cloud Shell, click on the Web Preview button and enter 3003 for the port.  If not using Cloud Shell, point a browser to `localhost:3000`.
 
-If not using Cloud Shell, point a browser to `localhost:3000`.
+The default username and password are both "admin".
 
 ## Destroy the TiDB cluster
 
 To destroy a TiDB cluster in Kubernetes, run the following command:
 
-{{< copyable "shell-regular" >}}
-
 ```shell
 kubectl delete tc basic -n demo
 ```
 
 To destroy the monitoring component, run the following command:
 
-{{< copyable "shell-regular" >}}
-
 ```shell
 kubectl delete tidbmonitor basic -n demo
 ```
 
 The above commands only delete the running pods, the data is persistent. If you do not need the data anymore, you should run the following commands to clean the data and the dynamically created persistent disks:
 
-{{< copyable "shell-regular" >}}
-
 ```shell
 kubectl delete pvc -n demo -l app.kubernetes.io/instance=basic,app.kubernetes.io/managed-by=tidb-operator && \
 kubectl get pv -l app.kubernetes.io/namespace=demo,app.kubernetes.io/managed-by=tidb-operator,app.kubernetes.io/instance=basic -o name | xargs -I {} kubectl patch {} -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
 ```
 
 ## Shut down the Kubernetes cluster
 
-Once you have finished experimenting, you can delete the Kubernetes cluster with:
-
-{{< copyable "shell-regular" >}}
+Once you have finished experimenting, you can delete the Kubernetes cluster:
 
 ```shell
 gcloud container clusters delete tidb
 ```
 
 ## More Information
 
-A simple [deployment based on Terraform] is also provided.
+To learn more about creating a deployment on GKE suitable for production use, please consult <https://pingcap.com/docs/tidb-in-kubernetes/stable/deploy-on-gcp-gke/>.
```

Comment:
```suggestion
To learn more about creating a deployment on GKE suitable for production use, please consult [Deploy TiDB on GCP GKE](deploy-on-gcp-gke.md).
```

Reply:
This document is loaded by Google Cloud Shell, so the URL has to point directly to the page on the PingCAP website. In other words, this document is not rendered by our website into web documentation).